### PR TITLE
Add rabbitmq

### PR DESCRIPTION
**Pull request by yotamazriel**

### CODE REVIEW

**Review by AsafYehezkelTL — APPROVED** [review]
LGTM

File: constructs/rabbitmq.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Construct } from 'constructs';
+import { Chart, Helm } from 'cdk8s';
+
+export class Rabbitmq extends Chart {
```
Comment (arikmaor) [inline constructs/rabbitmq.ts:4]:
```suggestion
export class RabbitMQ extends Chart {
```

**Review by arikmaor — APPROVED** [review]
LGTM

**Comment by yotamazriel** [comment]
After ruining the `npm test -- -u` locally the minio upgraded do we want to push it anyway?
@arikmaor

**Comment by arikmaor** [comment]
> After ruining the `npm test -- -u` locally the minio upgraded do we want to push it anyway?

It's hard for me to quite understand this sentence
IIUC, you're asking if you should run `git add --all` and `git push` after updating the snapshots. it that case, the answer is **YES**
If you're trying to ask something else please rephrase

**Comment by arikmaor** [comment]
@yotamazriel 
Great catch! `minio` needed a fixed version (I've pushed another commit to this branch)

BUT!

You still have a problem which should be addressed before merging
`rabbitmq` is generating some random cookie so `npm test` fails every time

**Comment by arikmaor** [comment]
@yotamazriel Set cookie secret like this to fix the issue:
```
      values: {
        auth: {
          user: 'user',
          password: '3e3abae2-6325-11ec-90d6-0242ac120003',
          erlangCookie: 'some random secret',
        },
      },
```

File: constructs/rabbitmq.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      },
+    });
+
+    new Helm(this, 'rabbitmq', {
+      chart: 'bitnami/rabbitmq',
+      values: {
+        auth: {
+          user: 'user',
+          password: '3e3abae2-6325-11ec-90d6-0242ac120003',
+          erlangCookie: 'some random secret',
```
Comment (arikmaor) [inline constructs/rabbitmq.ts:18]: I didn't actually mean this string...
not sure how spaces will be handled... (just generate a guid or something)

**Pull request merged**